activiti-cloud-modeling-service/activiti-cloud-starter-modeling/src/main/java/org/activiti/cloud/starter/modeling/configuration/ModelingSwaggerConfig.java

```diff
@@ -34,12 +35,17 @@
     }
 
     @Bean
-    public ApiInfo apiInfo(){
+    public ApiInfo apiInfo(BuildProperties buildProperties) {
         return new ApiInfoBuilder()
-                .title("Activiti Modeling REST API")
-                .description("Provides access to the core features of Activiti Modeling.")
-                .version("1.0")
-                .build();
+            .title(String.format("%s ReST API", buildProperties.getName()))
+            .description(buildProperties.get("description"))
+            .version(buildProperties.getVersion())
+            .license(String.format("© %s-%s %s. All rights reserved",
+                buildProperties.get("inceptionYear"),
+                buildProperties.get("year"),
+                buildProperties.get("organization.name")))
+            .termsOfServiceUrl(buildProperties.get("organization.url"))
+            .build();
```

Comment:
This code is the same for all the 4 starters. We could maybe move this somewhere inside `activiti-cloud-services-swagger` and keep only the bean instantiation here (if needed). What do you think?

Reply:
That's exactly what we are planning to do, but as part of a next pull request/story.